controller.go

```diff
@@ -786,13 +671,6 @@ func (c *Controller) syncHandler(key string) error {
 		break
 	}
 
-	// If an error occurs during Update, we'll requeue the item so we can
```

Comment:
What happened to this code?

Reply:
I moved it up as part of the refactoring.

In fact, this `err` is always nil in current master. IDE will report this `always nil` condition if you try to scan warnings in current master branch.

Also I already tested this scenario by disconnecting my network in the middle of executing. The requeue works fine after the changes.